phases/phase-1/challenges/README.md

```diff
@@ -1,8 +1,17 @@
 # Phase 1 challenges
-The first challenges of the Desmos Primer Program are divided into two levels: 
+The first challenges of the Desmos Primer Program are the following:
```

Comment:
The first challenges of the Desmos Primer Program are the following:

->

Phase 1 challenges include: